### PR TITLE
[MKC-1607] Updated cloud editor mentions

### PR DESCRIPTION
**Pull request by BenjaminDannegard**
## What This PR Changes
- Replaced web editor with cloud editor

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/software/ide-v1/tutorials/getting-started/cores/arduino-sam/installing-sam-core.md
Comment (jacobhylen) [inline content/software/ide-v1/tutorials/getting-started/cores/arduino-sam/installing-sam-core.md:1]: checked

File: content/software/ide-v2/tutorials/ide-v2-cloud-sketch-sync/ide-v2-cloud-sketch-sync.md
Comment (jacobhylen) [inline content/software/ide-v2/tutorials/ide-v2-cloud-sketch-sync/ide-v2-cloud-sketch-sync.md:1]: Checked

File: content/software/web-editor/act-on-multiple-sketches-at-once-on-the-web-editor/act-on-multiple-sketches-at-once-on-the-web-editor.md
Comment (jacobhylen) [inline content/software/web-editor/act-on-multiple-sketches-at-once-on-the-web-editor/act-on-multiple-sketches-at-once-on-the-web-editor.md:1]: checked

File: content/software/web-editor/getting-started-with-intel-based-platforms-on-arduino-create/getting-started-with-intel-based-platforms-on-arduino-create.md
Comment (jacobhylen) [inline content/software/web-editor/getting-started-with-intel-based-platforms-on-arduino-create/getting-started-with-intel-based-platforms-on-arduino-create.md:1]: vhecked

File: content/tutorials/generic/amplitude-serial-plotter/amplitude-serial-plotter.md
Comment (jacobhylen) [inline content/tutorials/generic/amplitude-serial-plotter/amplitude-serial-plotter.md:1]: checked

File: content/tutorials/generic/arduino-linux-on-up2-board-with-intel-mraa/arduino-linux-on-up2-board-with-intel-mraa.md
Comment (jacobhylen) [inline content/tutorials/generic/arduino-linux-on-up2-board-with-intel-mraa/arduino-linux-on-up2-board-with-intel-mraa.md:1]: checked

File: content/tutorials/generic/interacting-with-a-ti-sensortag-from-an-intel-nuc.md/interacting-with-a-ti-sensortag-from-an-intel-nuc.md.md
Comment (jacobhylen) [inline content/tutorials/generic/interacting-with-a-ti-sensortag-from-an-intel-nuc.md/interacting-with-a-ti-sensortag-from-an-intel-nuc.md.md:1]: checked

File: content/tutorials/generic/wave-playback/wave-playback.md
Comment (jacobhylen) [inline content/tutorials/generic/wave-playback/wave-playback.md:1]: checked

File: content/tutorials/generic/whistle-detector/whistle-detector.md
Comment (jacobhylen) [inline content/tutorials/generic/whistle-detector/whistle-detector.md:1]: checked

File: content/tutorials/projects/cancellinobot/cancellinobot.md
Comment (jacobhylen) [inline content/tutorials/projects/cancellinobot/cancellinobot.md:1]: checked

File: content/tutorials/projects/localize-your-board-with-an-sms/localize-your-board-with-an-sms.md
Comment (jacobhylen) [inline content/tutorials/projects/localize-your-board-with-an-sms/localize-your-board-with-an-sms.md:1]: checked

File: content/tutorials/projects/make-it-rain-clap-machine/make-it-rain-clap-machine.md
Comment (jacobhylen) [inline content/tutorials/projects/make-it-rain-clap-machine/make-it-rain-clap-machine.md:1]: checked

File: content/tutorials/projects/sd-sketch-update/sd-sketch-update.md
Comment (jacobhylen) [inline content/tutorials/projects/sd-sketch-update/sd-sketch-update.md:1]: checked

**Review by jacobhylen — COMMENTED** [review]
Review done from maker side.
@jcarolinares your turn :D

File: content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-breakout/datasheet/datasheet.md:1]: add entry log

File: content/hardware/04.pro/carriers/portenta-hat-carrier/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-hat-carrier/datasheet/datasheet.md:1]: add entry log

File: content/hardware/04.pro/carriers/portenta-hat-carrier/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-hat-carrier/tutorials/user-manual/content.md:1]: checked

File: content/hardware/04.pro/carriers/portenta-max-carrier/_unlisted/getting-started/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-max-carrier/_unlisted/getting-started/content.md:1]: checked

File: content/hardware/04.pro/carriers/portenta-max-carrier/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-max-carrier/datasheet/datasheet.md:1]: add entry level

File: content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/user-manual/content.md:1]: checked

File: content/hardware/04.pro/carriers/portenta-mid-carrier/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/carriers/portenta-mid-carrier/tutorials/user-manual/content.md:1]: checked

File: content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/tutorials/cheat-sheet/cheat-sheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-cat-m1-nb-iot-gnss-shield/tutorials/cheat-sheet/cheat-sheet.md:1]: checked

File: content/hardware/04.pro/shields/portenta-vision-shield/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-vision-shield/datasheet/datasheet.md:1]: add entry log

File: content/hardware/04.pro/shields/portenta-vision-shield/tutorials/connecting-to-ttn/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-vision-shield/tutorials/connecting-to-ttn/content.md:1]: checked

Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-vision-shield/tutorials/connecting-to-ttn/content.md:1]: checked

File: content/hardware/04.pro/shields/portenta-vision-shield/tutorials/ethernet-with-ide/content.md
Comment (jcarolinares) [inline content/hardware/04.pro/shields/portenta-vision-shield/tutorials/ethernet-with-ide/content.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev1/datasheet.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev1/datasheet.md:1]: add entry log

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev2/datasheet.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/datasheets/rev2/datasheet.md:1]: addd entry log

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system/content.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system/content.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md:1]: add entry log

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/pmc-arduino-library/content.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/pmc-arduino-library/content.md:1]: checked

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/user-manual/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-sense-me/datasheet/datasheet.md:1]: add entry log

File: content/hardware/06.nicla/boards/nicla-sense-me/tutorials/cheat-sheet/cheat-sheet.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-sense-me/tutorials/cheat-sheet/cheat-sheet.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-sense-me/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-sense-me/tutorials/user-manual/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-sense-me/tutorials/web-ble-dashboard/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-sense-me/tutorials/web-ble-dashboard/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md:1]: add entry log

File: content/hardware/06.nicla/boards/nicla-vision/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-vision/tutorials/user-manual/content.md:1]: checkec

File: content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/datasheet/datasheet.md:1]: dd entry level

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/ei-intruder-detector/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/getting-started-ml/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/getting-started-ml/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/motion-detection-ml/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/motion-detection-ml/content.md:1]: checked

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md
Comment (jcarolinares) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta-digital-exp/datasheet/datasheet.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta-digital-exp/datasheet/datasheet.md:1]: add entry log

File: content/hardware/07.opta/opta-family/opta/tutorials/01.user-manual/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/01.user-manual/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/02.getting-started-with-rs485/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/02.getting-started-with-rs485/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/03.getting-started-with-modbus-rtu/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/03.getting-started-with-modbus-rtu/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/04.getting-started-with-interrupts/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/04.getting-started-with-interrupts/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/08.memory-partitioning/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/08.memory-partitioning/content.md:1]: checked

File: content/hardware/07.opta/opta-family/opta/tutorials/13.energy-management-application-note/content.md
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tutorials/13.energy-management-application-note/content.md:1]: checked

**Review by jcarolinares — APPROVED** [review]
@jacobhylen done, as we said next time let's do it smaller, I think it will be great to all of us :) 

@BenjaminDannegard and @jacobhylen I think we should add the entry logs to those datasheets @TaddyHC can help with the pro ones so the overall additional work is more balanced, tell me what you think please

File: content/hardware/07.opta/opta-family/opta/tutorials/16.mac-address/content.md
Comment (jacobhylen) [inline content/hardware/07.opta/opta-family/opta/tutorials/16.mac-address/content.md:1]: @jcarolinares What about this one?

**Comment by BenjaminDannegard** [comment]
@jcarolinares if @TaddyHC can help with the PRO datasheets I can handle the MKR datasheets, then we can merge this massive PR :)

**Review by jacobhylen — APPROVED** [review]
Nice work.

**Pull request merged**